toolchain/sem_ir/formatter.cpp

```diff
@@ -772,11 +772,16 @@ class Formatter {
     out_ << InstT::Kind.ir_name();
     FormatInstructionRHS(inst);
     if (auto const_id = sem_ir_.constant_values().Get(inst_id);
-        const_id.is_constant()) {
-      out_ << (const_id.is_symbolic() ? " [symbolic" : " [template");
-      if (const_id.inst_id() != inst_id) {
-        out_ << " = ";
-        FormatInstName(const_id.inst_id());
+        !const_id.is_valid() || const_id.is_constant()) {
+      out_ << " [";
+      if (const_id.is_valid()) {
+        out_ << (const_id.is_symbolic() ? "symbolic" : "template");
+        if (const_id.inst_id() != inst_id) {
+          out_ << " = ";
+          FormatInstName(const_id.inst_id());
+        }
+      } else {
+        out_ << const_id;
```

Comment:
```suggestion
        out_ << "unknown";
```
Looking at how this appears for `import_ref`s, I think this might be clearer.

Reply:
I think the problem is that putting anything here for an ImportRefUnused is unclear: unused imports _never_ have a constant. How about this approach which special-cases ImportRefUnused to not print the constant?

(of course, now ImportRefUnused is very special-cased, but it feels like this should mirror ImportRefUsed where possible)